src/friendly-iframe-embed.js

```diff
@@ -228,13 +250,15 @@ export function installFriendlyIframeEmbed(
 
     // Add extensions.
     if (ampdoc && ampdocFieExperimentOn) {
-      extensions.installExtensionsInFie(
+      embed.installExtensionsInFie(
+        extensions,
         ampdoc,
         spec.extensionIds || [],
         opt_preinstallCallback
       );
     } else {
-      extensions.installExtensionsInChildWindow(
+      embed.installExtensionsInChildWindow(
```

Comment:
You can probably change these methods to `@private` now that they're only accessed from the same file.

Reply:
still used in tests though. I marked them @visibleForTesting . although I guess that annotation is currently a no-op other than documentation purpose?